CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - [#1249](https://github.com/JabRef/jabref/issues/1249) Date layout formatter added
 - Added ISBN integrity checker
 - Added filter to not show selected integrity checks
+- [#1225](https://github.com/JabRef/jabref/issues/1225): Hotkey-system should be consistent
```

Comment:
Where does one find information (help page / blog post / ...) about what was changed and how the hotkeys are configured now?